packages/wrangler/src/api/mixedMode/index.ts

```diff
@@ -15,7 +16,9 @@ export type MixedModeSession = Pick<Worker, "ready" | "dispose"> & {
 export async function startMixedModeSession(
 	bindings: BindingsOpt,
 	options?: {
-		auth: NonNullable<StartDevWorkerInput["dev"]>["auth"];
+		auth?: NonNullable<StartDevWorkerInput["dev"]>["auth"];
+		/** If running in a non-public compliance region, set this here. */
+		complianceRegion?: Config["compliance_region"];
```

Comment:
As we implement mixed mode we need to take this into account.
Clients (such as Wrangler and Vite plugin) will need to grab this property from any user config that they have loaded and pass it through.
cc @jamesopstad and @dario-piotrowicz

Reply:
Added to the local runtime controller. Didn't see this being called elsewhere...